sekoia_automation/cli.py

```diff
@@ -136,5 +137,13 @@ def openapi_to_module(
     ).run()
 
 
+@app.command(name="sync-library")
+def sync_library(
+    library: str = typer.Argument("../../modules", help="Path to the library"),
+    module: str = typer.Argument("", help="Module to deploy"),
+    check_image_on_registry: bool = typer.Option(False, help="Wether to check registry for existing image"),
+):
+    SyncLibrary(library, module, check_image_on_registry).execute()
```

Comment:
Keep naming of  other commands

```suggestion
    modules_path: Path = typer.Argument(..., help="Path to the playbook modules"),
    module: str = typer.Argument("", help="Module to deploy"),
    check_image_on_registry: bool = typer.Option(False, help="Wether to check registry for existing image"),
):
    SyncLibrary(modules_path, module, check_image_on_registry).execute()
```

Reply:
Are you sure we can take a Path from command line? typer will take charge of the conversion from the string input?